demo/transcribe_app.py

```diff
@@ -74,7 +104,21 @@ def transcribe(
     if isinstance(pipe, str):
         # Exception raised when loading
         return f"⚠️ Error: {pipe}"
-    text = pipe(audio)["text"]
+
+    output = pipe(
```

Comment:
Since we might process large audios, could we stream the inputs as they get processed?